src/dao/MoodDao.java

```diff
@@ -16,13 +17,20 @@
  */
 public class MoodDao {
 
+    Environment e = new Environment();
     ArrayList<Mood> m = new ArrayList<Mood>();
 
     /**
      * Constructs a new MoodDao
      */
     public MoodDao() throws DaoException {
         m = createMoods();
+        if(e.getEnvironment()== "development"){
```

Comment:
e.getEnvironment is returning null for the reasons I mentioned in a different comment. It causes the app to crash. To protect against that please try the following:

if (e.getEnvironment != null && e.getEnvironment().equals("development")